src/modifySongData.js

```diff
@@ -1,3 +1,10 @@
+// This is derived from DanceParty.getCurrentMeasure, but don't want to explicitly
+// depend on DanceParty so that this could be easily moved to a preRender step in
+// the future.
+function getMeasureForTime(songMetadata, time) {
+  return songMetadata.bpm * ((time - songMetadata.delay) / 240) + 1;
```

Comment:
I find the redundant parens actually make it harder to figure out why this math is the way it is... it took me a while to work out why we're using `240` here.  Might be worth this for readability:
```js
const measuresPerSecond = songMetadata.bpm / 240;
const secondsElapsed = time - songMetadata.delay;
return secondsElapsed * measuresPerSecond + 1;
```
I'd also document that the `time` parameter here is in seconds, not milliseconds, since it's less idiomatic for JavaScript.